src/components/Avatar/Avatar.vue

```diff
@@ -421,6 +421,12 @@ export default {
 		background-color: var(--color-text-maxcontrast);
 	}
 
+	&:not(&--unknown) {
+		// White background for avatars with transparency, also in dark theme
+		background-color: #fff !important;
```

Comment:
Why is the !important needed here?

Reply:
To overwrite the inline color that the avatar placeholder generates. Try removing it and it will look like the "before" images above.